ql/src/java/org/apache/hadoop/hive/ql/security/authorization/command/CommandAuthorizerV2.java

```diff
@@ -121,6 +142,32 @@ private static void addPermanentFunctionEntities(SessionState ss, List<ReadEntit
     return hivePrivobjs;
   }
 
+  /**
+   * A deferred authorization view is view created by non-super user like spark-user. This view contains a parameter "Authorized"
+   * set to false, so ranger will not authorize it during view creation. When a select statement is issued, then the ranger authorizes
+   * the under lying tables.
+   * @param Table t
+   * @return boolean value
+   */
+  private static boolean isDeferredAuthView(Table t){
+    String tableType = t.getTTable().getTableType();
+    String authorizedKeyword = "Authorized";
```

Comment:
What I meant was having a constant defined in HiveMetastoreAuthorizer.java like below.
public static final String DEFERRED_AUTHORIZED_KEY = "Authorized";

Once you do that you can use the key DEFERRED_AUTHORIZED_KEY everywhere whereever you are directly looking for "Authorized" key. The advantage of doing this way is that code is less error-prone and maintainable.
1. Future code modifications do introduce uppercase or lower case issues. (e.g. using "authorized" v/s "Authorized")
2. Its easy to look for all the places where deferred authorized key is being used in the IDE by looking for the usages of the constant. Currently, we will have to do a git grep "Authorized" which is inconvenient.